### PR TITLE
User Presence and Typing Indicators Code Snippet

### PR DESCRIPTION
**Pull request by zetbaitsu**
### Description

* [User Presence](https://getstream.io/chat/docs/presence_format/?language=kotlin) code snippet from web documentation.
* [Typing Indicators](https://getstream.io/chat/docs/typing_indicators/?language=kotlin) code snippet from web documentation. 

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~[ ] Changelog updated with client-facing changes~
- ~[ ] New code is covered by unit tests~
- ~[ ] Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/TypingIndicators.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        // Sends the typing.stop event
+        channelController.stopTyping().enqueue(result -> Unit.INSTANCE);
+    }
+
+    /**
+     * @see <a href="https://getstream.io/chat/docs/typing_indicators/?language=java#receiving-typing-indicator-events">Receiving Typing Events</a>
+     */
+    public void receivingTypingEvents() {
+        // Add typing start event handling
+        channelController.subscribeFor(
```
Comment (ogkuzmin) [inline stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/TypingIndicators.java:27]: poor Java programmers :-)

**Review by ogkuzmin — DISMISSED** [review]

File: stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/TypingIndicators.java (hunk 1 above)
Comment (zsmb13) [reply stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/TypingIndicators.java:27]: We support Java, but we're letting them know that we don't actually want them to write Java code 😄

File: stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/UserPresence.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    /**
+     * @see <a href="https://getstream.io/chat/docs/presence_format/?language=kotlin#listening-to-presence-changes">Listening to Presence Changes</a>
+     */
+    fun listeningPresenceChanges() {
+        // You need to be watching some channels/queries to be able to get presence events.
+        // There are multiple ways of doing so:
+        // 1. Watch a channel for presence event
+        val watchRequest = WatchChannelRequest()
+        watchRequest.presence = true
+        watchRequest.data["members"] = listOf("john", "jack")
```
Comment (adasiewiczr) [inline stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/UserPresence.kt:35]: NIT: We can use `apply` to set request's `presence` and `data`

File: stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/UserPresence.kt [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+import io.getstream.chat.docs.StaticInstances
+
+class UserPresence(val client: ChatClient, val channelController: ChannelClient) {
+
+    /**
+     * @see <a href="https://getstream.io/chat/docs/presence_format/?language=kotlin#invisible">Invisible</a>
+     */
+    fun invisible() {
+        val user = User("user-id")
+        user.invisible = true
```
Comment (adasiewiczr) [inline stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/UserPresence.kt:22]: NIT: It could be moved to `apply` block

**Review by adasiewiczr — APPROVED** [review]

**Pull request merged**